src/Sarif.Converters.UnitTests/TestData/FortifyFprConverter/ExpectedOutputs/OneResultBasic.sarif

```diff
@@ -11,7 +11,7 @@
       },
       "originalUriBaseIds": {
         "SRCROOT": {
-          "uri": "file:///C:/projects/myproject/src/"
+          "uri": "file://root/projects/myproject/src/"
```

Comment:
>/ [](start = 23, length = 1)

Again, doesn't `file:/root...` work? If not, don't bother to change it.

Reply:
Meaning keep it as a relative URI?

---
In reply to: [253102144](https://github.com/Microsoft/sarif-sdk/pull/1254#discussion_r253102144) [](ancestors = 253102144)

Reply:
I'm going to check in with this change, we can revert later after we discuss.

---
In reply to: [253125417](https://github.com/Microsoft/sarif-sdk/pull/1254#discussion_r253125417) [](ancestors = 253125417,253102144)